src/main/java/controller/user/UserLoginController.java

```diff
@@ -0,0 +1,69 @@
+package controller.user;
+
+import controller.AbstractController;
+import db.DataBase;
+import model.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import session.HttpSession;
+import session.HttpSessionRepository;
+import webserver.request.HttpRequest;
+import webserver.response.HttpResponse;
+import webserver.response.HttpStatus;
+import webserver.response.ResponseMetaData;
+
+import java.io.IOException;
+
+public class UserLoginController extends AbstractController {
+
+    private static final Logger log = LoggerFactory.getLogger(UserLoginController.class);
+
+    @Override
+    public void service(final HttpRequest request, final HttpResponse response) throws IOException {
+        String userId = request.findRequestBodyParam("userId");
+        String password = request.findRequestBodyParam("password");
+
+        log.debug("login request : userId={}", userId);
+
+        ResponseMetaData responseMetaData = buildFailedResponseMetaData(request);
+
+        if (DataBase.hasUser(userId)) {
```

Comment:
```suggestion
        User user = DataBase.findUserById(userId);
        if (user != null && user.matchPassword(password)) {
            log.debug("login success : userId={}", userId);
            HttpSession httpSession = createHttpSession();
            responseMetaData = buildSuccessfulResponseMetaData(request, httpSession.getId());
        }
```

이렇게하면 원뎁스로 줄일 수 있지 않을까요?